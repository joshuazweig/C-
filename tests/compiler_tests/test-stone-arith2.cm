int main () {
    stone s1;
    stone s2;

    s1 = "71823746";
    s2 = "13125867384543241324534";

    print_stone(s1);
    print_stone(s2);
}

