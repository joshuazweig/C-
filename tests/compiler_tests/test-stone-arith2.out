3731383233373436
3133313235383637333834353433323431333234353334
